.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+name: "Linting and Testing"
+
+on:
+  push:
+    branches: [master,github-actions-test]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+    - '.gitignore'
+
+  pull_request:
+    branches: [master]
+    paths-ignore:
+    - 'README.md'
+    - '.deepsource.toml'
+    - '.gitignore'
+
+
+jobs:
+  build:
+
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.6,3.7,3.8,3.9]
```

Comment:
We might want to drop python 3.6: I don't know how much it's used now?

Reply:
Good point, removed it.